docs/core/toc.yml

```diff
@@ -302,6 +304,8 @@ items:
       href: diagnostics/dotnet-trace.md
   - name: .NET Core diagnostics tutorials
     items:
+    - name: Measure event frequency with EventCounters
```

Comment:
Isn't this about more than event frequency?  How about something like "Get perf metrics with EventCounters"?